src/app/service/shoppingcart.service.ts

```diff
@@ -0,0 +1,32 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
+
+export class ShoppingCartService {
+  cart: any = [];
+
+  // getCart(){
+  //   return Promise.resolve(this.cart);
+  // };
+
+  addToCart(p_id: string, p_name: string, price: number, i_id: string, i_name: string) {
+
+    this.cart.push({
+      'product id': p_id,
+      'product name': p_name,
+      'price': Number(price),
+      'ingredient id': i_id,
+      'ingredient name': i_name
+    });
+    alert(`${p_name} added to cart`);
+    console.log(this.cart);
+  };
+
+// removeCart(searchId: string) {
```

Comment:
Are these comments supposed to be in the PR

Reply:
no, I will take them out